Lib/fontbakery/specifications/head.py

```diff
@@ -77,7 +77,7 @@ def parse_version_string(name):
 
   major, minor = version_string.group(1).split('.')
   major = str(int(major))  # "01.123" -> "1.123"
-  minor = minor.zfill(3)  # "3.0" -> "3.000", but "3.123" -> "3.123"
+  minor = minor.ljust(3, '0')  # "3.0" -> "3.000", but "3.123" -> "3.123"
```

Comment:
from a quick glance at the comments and tests, it looks to me this was done intentionally, so that `"1.3"` would be parsed as `("1", "003")`, and `"1.300"` would stay `("1", "300")`.

Reply:
oh I see. it's been also used to parse the head.fontRevision, which is a float, not only the nameID=5...
hm, this reminds me of https://github.com/fonttools/fonttools/issues/944#issuecomment-299973659